src/components/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import PropTypes from 'prop-types';
+
+function ComingSoon(props) {
```

Comment:
Destructure the props object in the argument. 
Like so: `function ComingSoon({ props })`